www/common/translations/messages.js

```diff
@@ -442,7 +442,7 @@ define(function () {
     out.fm_restoreDialog = "Are you sure you want to restore {0} to its previous location?";
     out.fm_unknownFolderError = "The selected or last visited directory no longer exist. Opening the parent folder...";
     out.fm_contextMenuError = "Unable to open the context menu for that element. If the problem persist, try to reload the page.";
-    out.fm_selectError = "Unable to select the targetted element. If the problem persist, try to reload the page.";
+    out.fm_selectError = "Unable to select the targeted element. If the problem persist, try to reload the page.";
```

Comment:
Now that I'm looking at this I also notice that it should be `persists`. I can fix this after merging this PR, I'm just commenting here so I don't forget.